test/scales/tickGeneratorsTests.ts

```diff
@@ -2,7 +2,7 @@
 
 describe("Scales", () => {
   describe("Tick generators", () => {
-    describe("Interval", () => {
+    describe("intervalTickGenerator()", () => {
```

Comment:
can we more semantically describe what this suite is for?

Maybe something like
"generating ticks with a given interval"?